### PR TITLE
change to sql file with prices / deleted ItemUser table

### PR DESCRIPTION
**Pull request by Ritin2000**
sql changes

### CODE REVIEW

File: database.sql [hunk 1]
```diff
@@ -18,416 +18,249 @@ USE `AuctionSite`;
...
-/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
-/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
-/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;
+DROP TABLE IF EXISTS Auction;
+CREATE TABLE Auction (
+	auctionName varchar(100) NOT NULL,
+    `description` varchar(1000)NOT NULL DEFAULT '',
+    minPriceCents decimal(15,2) NOT NULL, 
+    startPriceCents decimal(15,2) NOT NULL,
+    incPriceCents decimal(15,2) NOT NULL,
```
Comment (pn212) [inline database.sql]: Change name to dollar, not cents

File: database.sql [hunk 2]
```diff
@@ -18,416 +18,249 @@ USE `AuctionSite`;
...
+    itemId int NOT NULL,
+    auctionId int AUTO_INCREMENT PRIMARY KEY,
+    FOREIGN KEY (itemId) references Item (itemId)
+    
+) ;
+
+-- Table Structure for Bid -- 
+DROP TABLE IF EXISTS Bid;
+CREATE TABLE Bid (
+	amountCents int NOT NULL, -- price stored as total cents and later displayed as dollar amount --
```
Comment (pn212) [inline database.sql]: Change to decimal/dollars

File: database.sql [hunk 3]
```diff
@@ -18,416 +18,249 @@ USE `AuctionSite`;
...
+) ;
+
+-- Table Structure for Bid -- 
+DROP TABLE IF EXISTS Bid;
+CREATE TABLE Bid (
+	amount decimal(15,2) NOT NULL, -- price stored as total cents and later displayed as dollar amount --
+    bidDateTime datetime NOT NULL,
+    auctionId int NOT NULL,
+    userId int NOT NULL,
+    PRIMARY KEY (auctionId, amountCents),
```
Comment (pn212) [inline database.sql]: change key

File: database.sql [hunk 4]
```diff
@@ -18,416 +18,249 @@ USE `AuctionSite`;
...
+    itemId int NOT NULL,
+    auctionId int AUTO_INCREMENT PRIMARY KEY,
+    FOREIGN KEY (itemId) references Item (itemId)
+    
+) ;
+
+-- Table Structure for Bid -- 
+DROP TABLE IF EXISTS Bid;
+CREATE TABLE Bid (
+	amount decimal(15,2) NOT NULL, -- price stored as total cents and later displayed as dollar amount --
```
Comment (pn212) [inline database.sql]: change comment pls :)

Comment (Ritin2000) [reply database.sql]: :(

**Review by pn212 — APPROVED** [review]

**Pull request merged**